app/services/user_inviter.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+# Invite maker for new Users
+class UserInviter < ApplicationService
```

Comment:
you can't return member from this service, it doesn't clear from its name.
Use separate services